docs/element-definitions.md

```diff
@@ -1610,7 +1610,13 @@ series of bytes.
     - (Array Element) - Continuous bytes block. A pair of byte index and byte count.
         - `content` (array, fixed-type)
             - ([Number Element][]) - Zero-based index of a byte in the source document.
+                - attributes
+                    - line - The line number the source map starts on.
+                    - column - The colum number of the line that the source map starts on.
```

Comment:
typo: colum

Reply:
My dam `n` key is broke. Works 1/10 of the time 🤦‍♂️